app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,23 @@
+class MoviesController < ApplicationController
+  def index
+    @category   = params[:category] || 'title'
+    @sort_order = params[:sort_order] || 'asc'
+    @search     = params[:search]
+    @active     = 'movies'
+    if @category || @search
+      @library = Library.movies_search_sort(@search, @category, @sort_order)
```

Comment:
Count how many times you use the word "and" when answering this question: How would you describe what this method does?